tcms/core/management/commands/upgrade.py

```diff
@@ -0,0 +1,86 @@
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Perform post-upgrade tasks for Kiwi TCMS"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--noinput",
+            "--no-input",
+            "--yes",
+            "-y",
+            action="store_false",
+            dest="interactive",
+            help="Automatic mode. Does not require user confirmation",
+        )
+
+    def handle(self, *args, **kwargs):
+        output = None
+        if kwargs["verbosity"]:
+            output = self.stdout
```

Comment:
Output from this command should always be written to stdout. Verbosity should control the level of detail produced by the other commands which are being executed.